### PR TITLE
refactor: confirm dialog, multi language input, project-form, resource-form, list-info-form

### PR DESCRIPTION
**Pull request by derschnee68**
- Add ConfirmDialog for basic messages instead of DialogComponent
-  Add MultiLanguageInput and MultiLanguageTextarea instead of AppStringLiteral. Those two files follow formBuilder best practices.

- Add LoadingButtonDirective instead of long html for button < app-progresss-indicator < /button

- Add CommonInput, Chip List form components that follow formbuilder best practices.
- Add CreateProjectFormPage, EditProjectFormPage instead of ProjectForm (routing / refactor best practices)
- Refactor ResourceForm and ListInfoForm also With CreatePage and EditPage

### CODE REVIEW

**Review by irmastnt — CHANGES_REQUESTED** [review]

File: libs/vre/shared/app-string-literal/src/lib/multi-language-textarea.component.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    `,
+  ],
+})
+export class MultiLanguageTextareaComponent implements OnInit {
+  @Input() formGroup: FormGroup;
+  @Input() controlName: string;
+  @Input() editable = true;
+  @Input() placeholder: string;
+
+  constructor(public _myService: MultiLanguageFormService) {}
```
Comment (irmastnt) [inline libs/vre/shared/app-string-literal/src/lib/multi-language-textarea.component.ts:82]: bad naming "_myService"

File: apps/dsp-app/src/app/project/reusable-project-form/reusable-project-form.component.html
Comment (irmastnt) [inline apps/dsp-app/src/app/project/reusable-project-form/reusable-project-form.component.html:1]: remove this file

**Comment by irmastnt** [comment]
folder name: app-string-literal looks weird after refactoring. Probably need to change it or move components somewhere else.

**Comment by derschnee68** [comment]
> folder name: app-string-literal looks weird after refactoring. Probably need to change it or move components somewhere else.

Agree, but I am still in the middle of refactoring for this component. It will be done in next PR.

**Review by irmastnt — APPROVED** [review]

**Pull request merged**